src/index.html

```diff
@@ -4,9 +4,360 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet"
+    crossorigin="true">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+
+      <header class="header">
+        <a href="#" class="navbar-logo">
+          <img
+            class="logo_img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="navbar">
+          <ul class="navbar-list">
+            <li class="navbar-item">
+              <a 
+                href="#" 
+                class="is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">SAMSUNG</a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">SMARTPHONES</a>
+            </li>
+            <li class="navbar-item">
+              <a 
+                 href="#" 
+                 class="navbar-link" 
+                 data-qa="nav-hover"
+              >
+                LAPTOPS & COMPUTERS
+              </a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">GADGETS</a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">TABLETS</a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">PHOTO</a>
+            </li>
+            <li class="navbar-item">
+              <a href="#" class="navbar-link">VIDEO</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="main">
+        <div  class="card" data-qa="card">
+          <div class="card__top">
+            <img 
+              class="card__img" 
+              src="./images/imac.jpeg" 
+              alt="iMac"
+            >
+          </div>
+                    
+          <div class="card__bottom">
+            <div class="card__title">
+              <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+              <h3>Product code: 195434</h3>
+            </div>
+        
+            <div class="rating">
+              <div class="stars">
+                <div class="star"></div>
+                <div class="star"></div>
+                <div class="star"></div>
+                <div class="star"></div>
+                <div class="star5"></div>
+              </div>
```

Comment:
```suggestion
              <div class="stars stars--4">
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
              </div>
```

Reply:
use block stars from previous task `stars` 
you should have modifiers with number of active stars for stars block
(not hardcoding class stars5 with not active star)

Use modifiers

Reply:
still not resolved

![image](https://github.com/mate-academy/layout_catalog/assets/55200530/b137f07f-9962-4083-ad04-b200b64b7a17)